private/rules/maven_project_jar.bzl

```diff
@@ -133,6 +167,18 @@ single artifact that other teams can download and use.
             ],
             allow_empty = True,
         ),
+        "excluded_workspaces": attr.string_list(
+            doc = "A list of bazel workspace names to exclude from the generated jar",
+            allow_empty = True,
+            default = DEFAULT_EXCLUDED_WORKSPACES,
+        ),
+        "additional_dependencies": attr.label_keyed_string_dict(
+            doc = "Mapping of `Label`s to the excluded workspace names",
```

Comment:
This doc string isn't quite complete, mind mentioning that these deps should be found in the pom file?

Reply:
Expanded the comment.